src/gui/mzroll/pollyelmaveninterface.cpp

```diff
@@ -73,9 +74,9 @@ PollyElmavenInterfaceDialog::~PollyElmavenInterfaceDialog()
         delete (_loginform);
 }
 
-EPIWorkerThread::EPIWorkerThread()
+EPIWorkerThread::EPIWorkerThread(PollyIntegration* iPolly):
```

Comment:
The only point of `EPIWorkerThread` was to have a new `pollyIntegration` object for every process. This was introduced to handle parallel processing in different apps which is not an option anymore since we now block the UI when one app is running.

So ideally, `EPIWorkerThread` class should be removed, even though it shouldn't lead to any practical issues in the functionality. Just want to make sure you have taken this into consideration

Reply:
I did test it out and everything seemed to work fine.
We should get rid of `EPIWorkerThread` if it's not being used

Reply:
Yes. Can you add a TODO? We will take care of it in the next release.